cli/src/main/java/com/devonfw/tools/ide/io/IniFileImpl.java

```diff
@@ -0,0 +1,61 @@
+package com.devonfw.tools.ide.io;
+
+import java.util.LinkedHashMap;
+
+/**
+ * Implementation of {@link IniFile} preserves order of sections and properties between reading and writing
+ */
+public class IniFileImpl implements IniFile {
+
+  LinkedHashMap<String, IniSection> iniMap;
```

Comment:
https://github.com/devonfw/IDEasy/blob/main/documentation/coding-conventions.adoc#prefer-general-api
use `final` and `private` where possible.
```suggestion
  private final Map<String, IniSection> iniMap;
```